src/apps/inventory/views/Forms/Item/index.js

```diff
@@ -690,3 +682,18 @@ function PlannedLotView({ open, formState }) {
       </>
    )
 }
+
+function ContactPerson({ formState }) {
+   if (
+      formState.supplier?.contatcPerson &&
```

Comment:
```suggestion
      formState.supplier?.contactPerson &&
```

This typo would lead straight for falsy condition and none of it will render

Reply:
Aahh! didn't noticed that typo :(
moving this to a draft PR as there still are changes to be made